cluster/addons/fluentd-gcp/event-exporter.yaml

```diff
@@ -42,15 +42,15 @@ spec:
     metadata:
       labels:
         k8s-app: event-exporter
-        version: v0.1.9
+        version: v0.2.0
     spec:
       serviceAccountName: event-exporter-sa
       containers:
       - name: event-exporter
-        image: k8s.gcr.io/event-exporter:v0.1.9
+        image: k8s.gcr.io/event-exporter:v0.2.0
         command:
         - /event-exporter
-        - -sink-opts=-location={{ event_exporter_location }}
+        - -sink-opts=-stackdriver-resource-model={{ exporter_sd_resource_model }}
```

Comment:
It the location parameter no longer required? I understand why you add the new parameter, but it's not obvious why you're also removing the old one.